### PR TITLE
tern-subtitle-file-translator: Add version 1.1.1

### PR DESCRIPTION
**Pull request by wenmin92**
[Tern-Subtitle Fille Translator](https://tern.1c7.me/) is a .srt .ass .vtt file translator.

### CODE REVIEW

File: bucket/tern-subtitle-file-translator.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://tern.1c7.me",
+    "description": "Translate .srt .ass .vtt file",
```
Comment (Ash258) [inline bucket/tern-subtitle-file-translator.json]:
```suggestion
    "description": ".srt .ass .vtt file translator",
```

File: bucket/tern-subtitle-file-translator.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://tern.1c7.me/pricing"
+    },
+    "version": "1.1.1",
+    "url": "https://github.com/1c7/Translate-Subtitle-File/releases/download/1.1.1/win.zip",
```
Comment (Ash258) [inline bucket/tern-subtitle-file-translator.json]: Filename change in latest release is intentional?

`Windows-$version.zip` -> `win.zip`

File: bucket/tern-subtitle-file-translator.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://tern.1c7.me",
+    "description": "Translate .srt .ass .vtt file",
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://tern.1c7.me/pricing"
```
Comment (Ash258) [inline bucket/tern-subtitle-file-translator.json]: Where is service pricing mentioned?

https://doc.tern.1c7.me/folder/pricing/#there-are-2-pricing

Comment (wenmin92) [reply bucket/tern-subtitle-file-translator.json]: So, do I need to replace the link with https://doc.tern.1c7.me/folder/pricing/ ?

File: bucket/tern-subtitle-file-translator.json (hunk 2 above)
Comment (wenmin92) [reply bucket/tern-subtitle-file-translator.json]: I have communicated with the author, he will use a fixed name in the future, the file name of the latest version has been changed to `Tern_Windows_1.1.1.zip`.

File: bucket/tern-subtitle-file-translator.json (hunk 3 above)
Comment (Ash258) [reply bucket/tern-subtitle-file-translator.json]: Overall you need both. One for general usage of application (commercial, company, personal, ...) and services for unlimited usage.

Maybe notes about pricing of services would be suitable.

Comment (wenmin92) [reply bucket/tern-subtitle-file-translator.json]: OK, I will add a note.
I really admire your professionalism and dedication. 👍

File: bucket/tern-subtitle-file-translator.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            "Tern-Subtitle File Translator"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/1c7/Translate-Subtitle-File"
+    },
+    "autoupdate": {
+        "url": "https://github.com/1c7/Translate-Subtitle-File/releases/download/$version/Tern_Windows_$version.zip"
+    },
+    "notes": [
```
Comment (Ash258) [inline bucket/tern-subtitle-file-translator.json]:
```suggestion
    "notes": "Although application is 'Free forever', translator services are charged independently and based on usage (https://doc.tern.1c7.me/folder/pricing)"
```

**Pull request merged**